test/src/test/java/org/corfudb/runtime/view/ManagementViewTest.java

```diff
@@ -376,30 +377,42 @@ public void handleTransientFailure() throws Exception {
         failureDetected.acquire(2);
 
         log.info("Only allow SERVERS.PORT_0 to manage failures. Prevent the other servers from handling failures.");
-        TestRule testRule = new TestRule()
-                .matches(corfuMsg -> corfuMsg.getMsgType().equals(CorfuMsgType.SEAL)
-                        || corfuMsg.getMsgType().equals(CorfuMsgType.MANAGEMENT_FAILURE_DETECTED))
+
+        TestRule sealTestRule = new TestRule()
+                .requestMatches(msg -> msg.getPayload().getPayloadCase().equals(PayloadCase.SEAL_REQUEST))
+                .drop();
+
+        TestRule failureTestRule = new TestRule()
+                .matches(msg -> msg.getMsgType().equals(CorfuMsgType.MANAGEMENT_FAILURE_DETECTED))
                 .drop();
```

Comment:
Note: these were separated into two since it was awkward to implement a TestRule that was functional for both message formats. They can be recombined once all messages are sent with Protobuf.